src/test/java/io/aiven/kafka/connect/gcs/config/GcsSinkConfigTest.java

```diff
@@ -178,10 +181,7 @@ void unsupportedOutputField() {
         final Throwable t = assertThrows(
             ConfigException.class, () -> new GcsSinkConfig(properties)
         );
-        assertEquals("Invalid value [key, value, offset, timestamp, unsupported] "
-                + "for configuration format.output.fields: "
-                + "supported values are: 'key', 'value', 'offset', 'timestamp'",
-            t.getMessage());
+        assertThat(t.getMessage(), StringContains.containsString("supported values are:"));
```

Comment:
ahem, these changes are basically destroying the entire test.

Could they maybe be done in a little bit more subtle way? :) 

I admit, that original test is...questionable, but its destruction assures it won't be accepted in upstream, most likely.

Reply:
I will try to make pr to upstream like that - lets see